akka-actor/src/main/scala/akka/dispatch/ForkJoinExecutorConfigurator.scala

```diff
@@ -17,14 +18,25 @@ object ForkJoinExecutorConfigurator {
       parallelism: Int,
       threadFactory: ForkJoinPool.ForkJoinWorkerThreadFactory,
       unhandledExceptionHandler: Thread.UncaughtExceptionHandler,
-      asyncMode: Boolean)
-      extends ForkJoinPool(parallelism, threadFactory, unhandledExceptionHandler, asyncMode)
+      asyncMode: Boolean,
+      maxSpareThreads: Int)
+      extends ForkJoinPool(
+        parallelism,
+        threadFactory,
+        unhandledExceptionHandler,
+        asyncMode,
+        parallelism, // corePoolSize
```

Comment:
Previous constructor used 0 for this, from javadoc
> Using a smaller value (for example 0) has the same effect as the default.

Maybe just use that here as well, even if it means the same thing. To change as few things as possible.